packaging/conda/build.sh

```diff
@@ -101,11 +103,16 @@ then
   # identity and makes the build fail!
   CODESIGN="/usr/bin/codesign"
 
-  # Sign the MODL executable and check
+  # Obtain the path of the real KNI (not the symlinks)
+  KNI_PATH=$(file --mime --no-dereference $PREFIX/lib/libKhiops* | grep x-mach-binary | cut -d: -f1)
```

Comment:
I'm not sure I understand this well; some DLLs give us `x-sharedlib`. What do we want to achieve here?

Reply:
In macOS this is the output of the command
```bash
(kh-dev) folmos@MC-JWMHQQ2JYL:~/work/khiops/khiops (add-certs-to-conda)
(11:59:36)> file -I --no-dereference  ~/Downloads/khiops-conda-osx-arm64-4/osx-arm64/kni-10.3.0a.2-h2833df3_0/lib/libKhiopsNativeInterface.*
/Users/folmos/Downloads/khiops-conda-osx-arm64-4/osx-arm64/kni-10.3.0a.2-h2833df3_0/lib/libKhiopsNativeInterface.10.2.2-b.2.dylib: application/x-mach-binary; charset=binary
/Users/folmos/Downloads/khiops-conda-osx-arm64-4/osx-arm64/kni-10.3.0a.2-h2833df3_0/lib/libKhiopsNativeInterface.10.dylib:         inode/symlink; charset=binary
/Users/folmos/Downloads/khiops-conda-osx-arm64-4/osx-arm64/kni-10.3.0a.2-h2833df3_0/lib/libKhiopsNativeInterface.dylib:            inode/symlink; charset=binary
```

Reply:
Perhaps `readlink -f` might help achieve the goal here?

Reply:
I could change to `KNI_PATH=$(readlink -f $PREFIX/lib/libKhiops* | uniq)`

Reply:
Yeah, it seems more readable and straightforward IMHO.

Reply:
Perhaps add ` | head -n 1` to ensure that structurally there is a unique entry?

Reply:
Ok & Done.

Reply:
I don't see  the ` | head -n 1`...

Reply:
Oh, ok then the build would eventually fail because of this.